src/components/NewsletterSignup/NewsletterSignup.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import useStateWithDependencies from "../../hooks/useStateWithDependencies";
+import NewsletterSignup from "./NewsletterSignup";
+
+const meta: Meta<typeof NewsletterSignup> = {
+  title: "Components/Form Elements/NewsletterSignup",
+  component: NewsletterSignup,
+  decorators: [withDesign],
+  parameters: {
+    jest: ["NewsletterSignup.test.tsx"],
+  },
+  argTypes: {
+    className: { control: false },
+    id: { control: false },
+    confirmationText: {
+      control: "text",
+      table: {
+        defaultValue: {
+          summary:
+            "Thank you! You have successfully subscribed to our email updates! You can update your email subscription preferences at any time using the links at the bottom of the email.",
+        },
+      },
+    },
+    descriptionText: {
+      control: "text",
+      table: {
+        defaultValue: {
+          summary:
+            "Stay connected with the latest research news from NYPL, including information about our events, programs, exhibitions, and collections.",
+        },
```

Comment:
Yes it is intentional. My thinking is: why pass the default values as a demonstration since you would not do this IRL? So I opt to use the WithControls story as its own example of how one could use the component. I was picking up on some similar [practices](https://nypl.github.io/nypl-design-system/reservoir/v1/?path=/story/components-page-layout-table--with-controls) already in the DS.